test/local/metrics/amplitude.js

```diff
@@ -826,6 +831,128 @@ describe('metrics/amplitude', () => {
       });
     });
 
+    describe('email.verificationReminderFirstEmail.bounced', () => {
+      beforeEach(() => {
+        return amplitude('email.verificationReminderFirstEmail.bounced', mocks.mockRequest({}));
+      });
+
+      it('did not call log.error', () => {
+        assert.equal(log.error.callCount, 0);
+      });
+
+      it('called log.amplitudeEvent correctly', () => {
+        assert.equal(log.amplitudeEvent.callCount, 1);
+        const args = log.amplitudeEvent.args[0];
+        assert.equal(args[0].event_type, 'fxa_email - bounced');
+        assert.equal(args[0].event_properties.email_type, 'registration');
+      });
+    });
+
+    describe('email.verificationReminderFirstEmail.sent', () => {
+      beforeEach(() => {
+        return amplitude('email.verificationReminderFirstEmail.sent', mocks.mockRequest({}));
+      });
+
+      it('did not call log.error', () => {
+        assert.equal(log.error.callCount, 0);
+      });
+
+      it('called log.amplitudeEvent correctly', () => {
+        assert.equal(log.amplitudeEvent.callCount, 1);
+        const args = log.amplitudeEvent.args[0];
+        assert.equal(args[0].event_type, 'fxa_email - sent');
+        assert.equal(args[0].event_properties.email_type, 'registration');
+      });
+    });
+
+    describe('email.verificationReminderSecondEmail.bounced', () => {
+      beforeEach(() => {
+        return amplitude('email.verificationReminderSecondEmail.bounced', mocks.mockRequest({}));
+      });
+
+      it('did not call log.error', () => {
+        assert.equal(log.error.callCount, 0);
+      });
+
+      it('called log.amplitudeEvent correctly', () => {
+        assert.equal(log.amplitudeEvent.callCount, 1);
+        const args = log.amplitudeEvent.args[0];
+        assert.equal(args[0].event_type, 'fxa_email - bounced');
+        assert.equal(args[0].event_properties.email_type, 'registration');
```

Comment:
Should there be metadata stored as an event property saying which reminder bounced? It may be that one email type bounces more often than others and we'll want to know that.

Reply:
This reminds me, will bounces on verification reminders cause an [account to be deleted](https://github.com/mozilla/fxa-auth-server/blob/40b65eeb05a11437ae32cbf4b2ed06e30d71f261/lib/email/bounces.js#L154) like the initial signup verification email? My reading of the code is "yes, the account will be deleted if a verification reminder bounces".

Reply:
We include the template name on the event already. I'll add more assertions to make it clear.